### ISSUE TITLE
Can not install mooncoinWallet_64bit.zip

### ISSUE THREAD

**Issue by Logosferdi** [mooncoincore/wallet#9]
My E-mail: ferdiyao@gmail.com
Since July 24th, 2017, I downloaded the **mooncoinWallet_64bit.zip
![adresse portefeuille](https://user-images.githubusercontent.com/29759285/34653257-5e645484-f3e1-11e7-8269-c53836fb2aef.jpg)
![hash transaction](https://user-images.githubusercontent.com/29759285/34653258-5e910812-f3e1-11e7-8a26-724cf1abbd3a.jpg)

** and at the moment the synchronization had started, I made a purchase of **275,409.16 Mooncoin**.
The next day, I clicked on the wallet to continue the synchronization, my controller noted that an error has occurred so that the program can not run.
Since then, I have made every effort to restore my wallet and get my mooncoin but without success.
Finally I come by this email to contact you if there are prior arrangements to make before installing the wallet.
Transaction Hash is:  **540c84e71230464a4920f7f5ed69ef3bc98c957d61a9e4518af8b48569848633**
My Mooncoin adresse is: **2LgvfwuBzYobCTqdy6ZhoXR7A6sH98wzhM**
I downloaded the wallet several times but could not run the program.
Is it a general problem or is it with my computer?
I really need your solutions please.
In conclusion, I ask you to help me please!
Thank you

**Comment by mooncoincore** [comment]
Sorry for the delay! I'm under pressure the last days.
Please from your mooncoin directory(location C:\Users\yourusername\AppData\Roaming\ i think except if you changed it) remove all the files except your wallet.dat.
(Always protect your wallet.dat and your private keys)
Then re-run the wallet and let it to do a new full sync.

**Comment by Logosferdi** [comment]
I made a mistake, I do not have a private key. When the synchronization
started, I then looked for the address of the wallet and I bought the
Mooncoin coins by communicating the address to the seller and he
transferred my coins. The next day, I turned on the computer to continue
the synchronization but since then, nothing works.
Is it possible to send me an installable Mooncoin file?

N'goran Ferdinand YAO
Administrateur Principal des Services Financiers
Contrôleur des Assurances
Whatsapp : +22507361245 <+225%2007%2036%2012%2045>
Skype : ferdiyao@gmail.com
Contact: +225 07 36 12 45 <+225%2007%2036%2012%2045>
               +225 06 32 77 49 <+225%2006%2032%2077%2049>
               *+225 53 50 33 33*

2018-01-14 12:38 GMT+00:00 MoonCoinCore <notifications@github.com>:

**Comment by killmode** [comment]
If you are yet to uninstall the original version, you can go to the program directory and look for the .wallet file which you open with a text editor and copy the information therein and store is a private place for safe keeping and future reference.... Else if you don't have that, there may be nothing you can do... Simply means you've lost your token(but I hope  you don't).

**Comment by mebagger** [comment]
maybe your computer is not 64 bit. Try the 32bit version.
